applications/PfemFluidDynamicsApplication/custom_processes/set_material_properties_from_fluid_to_rigid_nodes_process.hpp

```diff
@@ -133,19 +133,19 @@ namespace Kratos
           bulk_modulus = elemProperties[BULK_MODULUS];
           viscosity = elemProperties[DYNAMIC_VISCOSITY];
 
-          if (elemProperties.Has(YIELD_SHEAR)) //Bingham model
+          if (elemProperties.Has(YIELD_SHEAR)) // Bingham model
           {
             flow_index = elemProperties[FLOW_INDEX];
```

Comment:
you checked Yield but this variable has not been checked, can be dangeorus

Reply:
The rest of variables are present if there is Yield_shear. They are required by the Bingham model